gcp/table_gcp_compute_tpu.go

```diff
@@ -0,0 +1,252 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+
+	tpu "google.golang.org/api/tpu/v2"
+)
+
+const matrixKeyZone = "zone"
+
+func tableGcpComputeTpu(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_compute_tpu",
```

Comment:
Willit be better to rename the table to `gcp_tpu_v2_vm` I could see terraform named it as [google_tpu_v2_vm](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/tpu_node)